libs/util/data-fetcher/src/parsers/base.ts

```diff
@@ -1,15 +1,27 @@
-import { DataItem, DatasetInfo, PropertyInfo } from '../lib/model'
-import { fetchData } from '../lib/utils'
+import {
+  DataItem,
+  DatasetInfo,
+  FieldAggregation,
+  FieldFilter,
+  FieldGroupBy,
+  FieldName,
+  FieldSort,
+  PropertyInfo,
+} from '../lib/model'
 
 export class BaseDataset {
```

Comment:
you're right, it's not very clear; how about we use the concept of "readers", with `BaseReader`, `BaseFileReader`, `CsvReader` etc... then the `parsers` folder chould be renamed to `readers`?

Reply:
Reader could do it, Querier ? idk we have to really understand what it's meant to do
- fetch the data ?
- open a connection
- select
- filter
- sort
- aggregate
:thinking:

Reply:
basically the general mission of the data-fetcher is to extract data from various sources and serve it in a common format (DataItems). The readers are then responsible for implementing the various possibilities depending on what the source allows. It could be HTTP queries, SQL queries, GraphQL or whatever else.